core/common/lib/transform-lib/src/test/java/org/eclipse/edc/transform/transformer/edc/from/JsonObjectFromSecretTransformerTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2024 Amadeus
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Amadeus - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.transform.transformer.edc.from;
+
+
+import jakarta.json.Json;
+import jakarta.json.JsonBuilderFactory;
+import org.eclipse.edc.spi.types.domain.secret.Secret;
+import org.eclipse.edc.transform.spi.TransformerContext;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.edc.jsonld.spi.JsonLdKeywords.TYPE;
+import static org.eclipse.edc.spi.types.domain.secret.Secret.EDC_SECRET_VALUE;
+import static org.eclipse.edc.spi.types.domain.secret.Secret.PROPERTY_ID;
+import static org.mockito.Mockito.mock;
+
+
+class JsonObjectFromSecretTransformerTest {
+
+    private final JsonBuilderFactory jsonFactory = Json.createBuilderFactory(Map.of());
+    private final TransformerContext context = mock();
+
+    private final JsonObjectFromSecretTransformer transformer = new JsonObjectFromSecretTransformer(jsonFactory);
+
+    @Test
+    void transform_returnJsonObject() {
```

Comment:
please add some negative tests, like when the secret id is missing, etc. Generally, transformers should not have any validation, as that is handled elsewhere, so we expect null values then (as opposed to: exceptions)

Reply:
Not sure we add a lot here.
If we forget the `id` then we wont even be able to create a `Secret` and call the function. 
If we forget the `value` then nothing special will happen.